gallery/src/styles/NxList/NxListPage.tsx

```diff
@@ -124,6 +124,34 @@ const NxListPage = () =>
               would only be a single list item which contains the error alert.
             </td>
           </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-list__item--clickable</code></td>
```

Comment:
I feel like either this class or `.nx-list--clickable` shouldn't be necessary.  One or the other should be enough to provide the necessary styling.

Reply:
> I feel like either this class or `.nx-list--clickable` shouldn't be necessary. One or the other should be enough to provide the necessary styling.

~The deprecated clickable lists require `.nx-list--clickable` to differentiate the <li>’s from the non-clickable ones, and we need `.nx-list__item--link` and `.nx-list__item--clickable` to differentiate those <li>’s from the deprecated ones.~

I figured out a solution, though in some respects I like the intentionality of adding `nx-list--clickable` to lists. https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/a24817c3fb8609eb0f60d86509a1bb46fc1c99e0

Reply:
> I figured out a solution, though in some respects I like the intentionality of adding nx-list--clickable to lists. a24817c
>
> WDYT?

I like it